### PR TITLE
fix: use buzzpay new URL format

### PR DESCRIPTION
**Pull request by rolznz**
Fixes https://github.com/getAlby/hub/issues/759

`name` param is not read correctly and needs fixing on the PoS side.

### CODE REVIEW

**Comment by bumi** [comment]
I am not sure if I can fix the query param on the POS side. maybe I ask some AI :) 

should we use `name` or `label` - I think internally we call it label there.

**Comment by rolznz** [comment]
@bumi added here: https://github.com/getAlby/pos/commit/86363cf55eae67bbe74664be3643885edd36873d also to the share link: https://github.com/getAlby/pos/commit/08ae31c6fbb4054974d70d33c2d2838101bade89

File: frontend/src/screens/internal-apps/BuzzPay.tsx [hunk 1]
```diff
@@ -51,7 +51,7 @@ export function BuzzPay() {
         });
 
         setPosUrl(
-          `https://pos.albylabs.com/?name=${encodeURIComponent(name)}#/wallet/${encodeURIComponent(createAppResponse.pairingUri)}/new`
+          `https://pos.albylabs.com/#/?nwc=${btoa(createAppResponse.pairingUri)}&name=${encodeURIComponent(name)}`
```
Comment (rolznz) [inline frontend/src/screens/internal-apps/BuzzPay.tsx:54]:
```suggestion
          `https://pos.albylabs.com/#/?nwc=${btoa(createAppResponse.pairingUri)}&label=${encodeURIComponent(name)}`
```

**Comment by rolznz** [comment]
> should we use name or label - I think internally we call it label there.

It supports both, but I agree it makes more sense as label. I've updated it.

**Comment by bumi** [comment]
🚀

**Pull request merged**